### ISSUE TITLE
2.3.0-beta.2 - asset references not checksumed in index.html

### ISSUE THREAD

**Issue by jackmatt2** [ember-cli/ember-cli#5515]
```
$ ember --version

version: 2.3.0-beta.2
node: 0.12.3
npm: 2.14.10
os: linux x64

$ ember build --environment=production
```

assets in index.html are not checksummed.
```
$ cd dist
$ vi index.html

    ...
    <script src="assets/vendor.js"></script>
    <script src="assets/myproject.js"></script>
```

However assets are correctly checksummed in /assets folder.
```
$ cd assets
$ ls

...
app.scss                                         
myproject-bc73f524ec4c3af2dacf4e0aee9f18d2.css                     
vendor-8bf40b5b5c068344e1c07e1b4e8852fb.js
myproject-5ade133510a6795a212ca284fd5c7042.js  
ember-basic-dropdown.scss                         
ember-power-select  vendor-656ca136e28cb3e0da908f11b479a2d2.css
```

**Comment by jackmatt2** [comment]
Hang on - just realised that 2.3.0 is released - let me give that a try.

**Comment by jackmatt2** [comment]
Verified to be fixed in 2.3.0
